### DISCUSSION TITLE
Yi-9b 提示词的英文比中文强吗

### DISCUSSION THREAD

**Discussion by libingbingd** [01-ai/Yi#455] [Q&A] [answered]
Yi-9b 提示词的英文比中文强吗？
在体验的时候，同样的提示，中文输出的结果乱七八糟，把中文翻译成英文，输出就正确；

**Comment by itsliupeng** [comment]
都没有针对性优化，作为 base 模型，可以将有问题的 prompt 修改为更为明确详细的版本试试
另外，如果是用 gguf q4 的版本，可能不如 bf16 的更好

**Comment by itsliupeng** [comment, marked as answer]
另外，对于base模型，最好用 in-context learning 的方式方式来测试，类似测评的 few-shot 而不是 0-shot
给出一个示例，再去问，体感效果会比较好
